config/samples/backends/ceph/backend.yaml

```diff
@@ -0,0 +1,61 @@
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  extraMounts:
+    - name: v1
+      region: r1
+      extraVol:
+        - propagation:
+          - CinderVolume
+          - CinderBackup
+          - GlanceAPI
+          extraVolType: Ceph
+          volumes:
+          - name: ceph
+            projected:
+              sources:
+              - secret:
+                  name: ceph-conf-files
+          mounts:
+          - name: ceph
+            mountPath: "/etc/ceph"
+            readOnly: true
+  cinder:
+    template:
+      cinderVolumes:
+        volume1:
```

Comment:
Per my previous comment, it would be nice if this c-vol could be named "ceph" instead of "volume1"

Reply:
Update: Per my update above, by eliminating the entire cinderVolumes section in the base CR, I was able to choose a name  like "ceph" here, when the c-vol pod is added.

Reply:
I was doing it for the same reason as above. Changing it now.